be1-go/channel/digitalCash/mod_test.go

```diff
@@ -235,6 +235,63 @@ func Test_SendReaction(t *testing.T) {
 	require.NoError(t, channel.Publish(message, socket.ClientSocket{}))
 }
 
+// Tests that the channel works correctly when it receives a large transaction
+func Test_SendTransactionMaxAmount(t *testing.T) {
+	// Create the hub
+
+	var laoID = "fzJSZjKf-2cbXH7kds9H8NORuuFIRLkevJlN7qQemjo="
+	var sender = "M5ZychEi5rwm22FjwjNuljL1qMJWD2sE7oX9fcHNMDU="
+	var digitalCashChannelName = "/root/" + laoID + "/digitalCash/transaction"
```

Comment:
```suggestion
	var digitalCashChannelName = "/root/" + laoID + "/coin"
```
Update according to the specification

Reply:
I can put it here but it would make more sense to put it in #938

Reply:
Hold on my bad I misread the diff